standalone-metastore/metastore-server/src/main/resources/package.jdo

```diff
@@ -1601,6 +1609,41 @@
         <column name="DB_ID"/>
       </index>
     </class>
+    <class name="MDataConnector" table="DATACONNECTORS" identity-type="application" detachable="true">
+       <field name="name" primary-key="true">
+         <column name="NAME" length="128" jdbc-type="VARCHAR" allows-null="false"/>
+       </field>
+       <field name="type">
+         <column name="TYPE" length="128" jdbc-type="VARCHAR" allows-null="false"/>
+       </field>
+       <field name="url">
+         <column name="URL" length="4000" jdbc-type="VARCHAR" allows-null="false"/>
+       </field>
+       <field name="description">
+         <column name="COMMENT" length="256" jdbc-type="VARCHAR"/>
+       </field>
+       <field name="ownerName">
+         <column name="OWNER_NAME" length="128" jdbc-type="VARCHAR" allows-null="true"/>
+       </field>
+       <field name="ownerType">
+         <column name="OWNER_TYPE" length="10" jdbc-type="VARCHAR" allows-null="true"/>
+       </field>
+       <field name="createTime">
+         <column name="CREATE_TIME" jdbc-type="integer" allows-null="false"/>
+       </field>
+       <field name="parameters" table="DATACONNECTOR_PARAMS">
+         <map key-type="java.lang.String" value-type="java.lang.String"/>
+          <join>
+            <column name="NAME"/>
+          </join>
+          <key>
+            <column name="PARAM_KEY" length="180" jdbc-type="VARCHAR"/>
+          </key>
+          <value>
+           <column name="PARAM_VALUE" length="32672" jdbc-type="VARCHAR"/>
```

Comment:
Just confirming, Should this be 32762?

Reply:
you are right, but there are 5 other occurence in this file that are set to the value I set to MTable, MSerdeInfo, MStorageDescriptor etc
but I dont think it actually matters because JDO uses a String to map these values that has a max of 2GB. In the HMS schema, the TABLE_PARAMS.PARAM_VALUE is a CLOB. and it works fine.